packages/next/src/client/script.tsx

```diff
@@ -352,6 +352,10 @@ function Script(props: ScriptProps): JSX.Element | null {
           />
         )
       }
+    } else if (strategy === 'worker') {
+      return (
+        <script nonce={nonce} type="text/partytown" {...restProps} id={id} />
```

Comment:
We can remove this note in the docs https://github.com/vercel/next.js/blob/canary/docs/02-app/01-building-your-application/06-optimizing/05-scripts.mdx#offloading-scripts-to-a-web-worker-experimental